e2e/specs/menu_bar/file_menu.test.js

```diff
@@ -39,4 +39,21 @@ describe('file_menu/dropdown', function desc() {
             settingsWindow.should.not.be.null;
         }
     });
+
+    it('MM-T805 Sign in to Another Server Window opens using menu item', async () => {
+        if (process.platform === 'win32' || process.platform === 'linux') {
+            const mainWindow = this.app.windows().find((window) => window.url().includes('index'));
+            mainWindow.should.not.be.null;
+            robot.keyTap('alt');
+            robot.keyTap('enter');
+            robot.keyTap('f');
+            robot.keyTap('s');
+            robot.keyTap('s');
```

Comment:
Is the second `s` just to make sure it works?

Reply:
No @devinbinnie The second s is to take the cursor to sign in to another server tab, first s takes it to settings.

Reply:
Ah, right :P